packages/blade/src/storybook-recipes/BladeForm.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import type { Meta } from '@storybook/react';
+import { RecipeSandbox } from '~src/_helpers/storybook/Sandbox';
+
+export function BladeForm(): JSX.Element {
+  return (
+    <RecipeSandbox
+      title="Blade Form"
+      codesandboxId="blade-form-7holu5"
+      activeFile="/src/Form.tsx"
+    />
+  );
+}
+
+export default {
+  title: 'Recipes/Blade Form',
```

Comment:
Instead of Blade Form what if we name it as Simple Form?

Reply:
done